src/main/java/org/mqttbee/api/mqtt/mqtt5/message/publish/Mqtt5PublishBuilder.java

```diff
@@ -182,10 +183,8 @@ public Mqtt5PublishBuilder(@Nullable final Function<? super Mqtt5Publish, P> par
     }
 
     @NotNull
-    public Mqtt5PublishBuilder<P> topicAliasUsage(@NotNull final TopicAliasUsage topicAliasUsage) {
-        Preconditions.checkArgument(
-                topicAliasUsage == MUST_NOT || topicAliasUsage == MAY || topicAliasUsage == MAY_OVERWRITE);
-        this.topicAliasUsage = topicAliasUsage;
+    public Mqtt5PublishBuilder<P> useTopicAlias(@NotNull final TopicAliasUsage useTopicAlias) {
```

Comment:
Minor: the attribute and parameter should stay topicAliasUsage

Reply:
DONE